feedback/templates/feedback_tags/contexttag_confirm_delete.html

```diff
@@ -0,0 +1,22 @@
+{% extends "manage/base.html" %}
+{% load i18n %}
+{% load contexttag %}
+
+{% block content %}
+	<h1>{% trans "Confirm feedback tag removal" %}</h1>
```

Comment:
Django started to recommend `translate` instead of `trans` so that it doesn't sound like transgender.